src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -556,6 +556,16 @@ def edit_user(conn, username, bank=None, **kwargs):
                     clear_queues(conn, username, bank)
                 elif field == "projects":
                     clear_projects(conn, username, bank)
+                elif field in ("max_nodes", "max_cores"):
+                    # set value to max integer
+                    update_stmt = f"UPDATE association_table SET {field}=2147483647 WHERE username=?"
+                    tup = (username,)
```

Comment:
Another place where I feel like it would be clearer to have a named constant rather than the integer directly.